snooty/test_parser.py

```diff
@@ -123,24 +121,27 @@ def test_literalinclude() -> None:
     )
     page.finish(diagnostics)
     assert diagnostics == []
-    assert ast_to_testing_string(page.ast) == "".join(
-        (
-            "<root>",
-            '<code lang="py">db.inventory.insert_many([\n',
-            '    {"item": "journal",\n',
-            '     "qty": 25,\n',
-            '     "tags": ["blank", "red"],\n',
-            '     "size": {"h": 14, "w": 21, "uom": "cm"}},\n\n',
-            '    {"item": "mat",\n',
-            '     "qty": 85,\n',
-            '     "tags": ["gray"],\n',
-            '     "size": {"h": 27.9, "w": 35.5, "uom": "cm"}},\n\n',
-            '    {"item": "mousepad",\n',
-            '     "qty": 25,\n',
-            '     "tags": ["gel", "blue"],\n',
-            '     "size": {"h": 19, "w": 22.85, "uom": "cm"}}])</code>',
-            "</root>",
+    check_ast_testing_string(
+        page.ast,
+        "<root>"
+        + "".join(
```

Comment:
Why does this particular string require `.join()`? Is this something that is necessary with `<code>`?

Reply:
Updated!